### PR TITLE
(DOCSP-26500): Add Set the Client Log Level - Kotlin SDK page

### PR DESCRIPTION
**Pull request by cbullinger**
### Jira

- https://jira.mongodb.org/browse/DOCSP-26500

### Staged Changes

- [Set the Client Log Level - Kotlin SDK](https://docs-mongodbcom-staging.corp.mongodb.com/realm/docsworker-xlarge/docsp-26500-log-level/sdk/kotlin/sync/log-level/) - new page

### Reminder Checklist

If your PR modifies the docs, you might need to also update some corresponding
pages. Check if completed or N/A.

- [x] Create Jira ticket for corresponding docs-app-services update(s), if any
- [x] Checked/updated Admin API
- [x] Checked/updated CLI reference

### Review Guidelines

[REVIEWING.md](https://github.com/mongodb/docs-realm/blob/master/REVIEWING.md)

### CODE REVIEW

File: examples/kotlin/shared/src/commonTest/kotlin/com/mongodb/realm/realmkmmapp/SyncTest.kt [hunk 1]
```diff
@@ -336,5 +336,63 @@ class SyncTest: RealmTest() {
...
+            // Access your app
+            val app = App.create(yourFlexAppId)
+            val user = app.login(credentials)
+
+            // Access the configuration builder for the app
+            val config = SyncConfiguration.Builder(user, setOf(Toad::class))
+
+                // Set the custom logger and log level
+                // Must be set BEFORE you open a synced realm
+                .log(LogLevel.ALL, customLoggers = listOf(customLogger))
```
Comment (dacharyc) [inline examples/kotlin/shared/src/commonTest/kotlin/com/mongodb/realm/realmkmmapp/SyncTest.kt:383]: There's no example of what this `customLogger` might look like, and I didn't see a link to anything describing what this implementation should be. It looks like `customLoggers` is a list of RealmLogger objects, so I wonder if we want to link to [RealmLogger](https://www.mongodb.com/docs/realm-sdks/kotlin/latest/library-base/-realm%20-kotlin%20-s-d-k/io.realm.kotlin.log/-realm-logger/index.html) copy that goes along with this example. Additionally, I wonder if we want to provide just a basic placeholder `customLogger` to show what that might look like, similar to [the Swift example](https://github.com/mongodb/docs-realm/blob/272fe781ac6d65e21eefc7b54879b1d321faec87/examples/ios/Examples/Sync.swift#L364). (Just showing that this exposes a LogLevel and a String message, plus whatever additional stuff you want to throw in.)

**Review by dacharyc — CHANGES_REQUESTED** [review]
We should tweak the "enum" wording because it's not quite right, and I also wonder if we want to elaborate on the custom logger example, or at least add a link to the API reference.

File: examples/kotlin/shared/src/commonTest/kotlin/com/mongodb/realm/realmkmmapp/SyncTest.kt (hunk 1 above)
Comment (cbullinger) [reply examples/kotlin/shared/src/commonTest/kotlin/com/mongodb/realm/realmkmmapp/SyncTest.kt:399]: Updated to include link to RealmLogger docs and example to better match Swift (sdk engg didn't think it necessary to include more than that -- e.g. full custom logger implementation/logic)

File: source/sdk/kotlin/sync/log-level.txt [hunk 2]
```diff
@@ -0,0 +1,49 @@
+.. _kotlin-client-log-level:
+
+=====================================
+Set the Client Log Level - Kotlin SDK
```
Comment (dacharyc) [inline source/sdk/kotlin/sync/log-level.txt:4]: I notice in the ToC we call this "Set the Sync Log Level" - should we update the title here to match?

**Review by dacharyc — APPROVED** [review]
Overall LGTM, just a Q about the page title and the ToC entry.

File: source/sdk/kotlin/sync/log-level.txt (hunk 2 above)
Comment (cbullinger) [reply source/sdk/kotlin/sync/log-level.txt:4]: Was on purpose, but I changed TOC entry to match page title

**Pull request merged**